templates/shared_services/gitea/terraform/variables.tf

```diff
@@ -47,8 +62,33 @@ variable "gitea_storage_limit" {
   default     = 1024
 }
 
+variable "log_analytics_workspace_id" {
+  type        = string
+  description = "ID of the Log Analytics workspace for TRE"
+}
+
+variable "core_app_service_plan_id" {
+  type        = string
+  description = "Name of the App Service plan"
+}
+
+variable "core_application_insights_instrumentation_key" {
+  type        = string
+  description = "Instrumentation key for the Core Application Insights"
+}
+
 variable "gitea_allowed_fqdns" {
   type        = string
   description = "comma seperated string of allowed FQDNs for Gitea"
   default     = "github.com, www.github.com, api.github.com, git-lfs.github.com, *githubusercontent.com"
 }
+
+variable "firewall_name" {
+  type        = string
+  description = "Name of the firewall to connect to"
+}
+
+variable "firewall_resource_group_name" {
+  type        = string
+  description = "Name of the firewall to connect to"
```

Comment:
Name of the resource group containing the firewall resource

Reply:
Thanks for spotting this :)